src/poker.js

```diff
@@ -1,10 +1,104 @@
-function winningPair() {}
+function isPair(pair) {
+  return pair[0] === pair[1]
+}
+
+function symbolNumber(symbol) {
+  // find numeric representation of symbol
+  switch (symbol) {
+    case 'J':
+      return 11
+    case 'Q':
+      return 12
+    case 'K':
+      return 13
+    case 'A':
+      return 14
+    default:
+      return parseInt(symbol)
+  }
+}
```

Comment:
Nice work on this exercise. Another way you could define card values is by using a standard object as a map:

```js
const scores = {
'A': 14,
'K': 13,
'Q': 12,
'J': 11,
'10': 10,
...
}

const cardScore = scores['Q']
```